README.md

```diff
@@ -3,13 +3,75 @@
 [![Tests](https://github.com/lidofinance/lido-oracle/workflows/Tests/badge.svg?branch=daemon_v2)](https://github.com/lidofinance/lido-oracle/actions)
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 
+Oracle daemon for Lido decentralized staking service. Collects and reports CL states to the Lido dApp contracts running on Ethereum EL side.
+
+There are two types of oracles:
+  - **accounting:**
+      reports info about validators count, balances (validators, withdrawal vault, EL rewards vault), count of exited validators
+  - **ejector:**
+      reports info about next validators to be ejected (to initiate exit from CL)
+
+# Usage
+
+## Setup
+Oracle daemon must be run using a docker container. Images is available on [Docker Hub](https://hub.docker.com/r/lidofinance/oracle).
+Pull the image using the following command:
+```bash
+docker pull lidofinance/oracle:{tag}
+```
+Where `{tag}` is a version of the image. You can find the latest version in the [releases](https://github.com/lidofinance/lido-oracle/releases)
+**OR**\
+You can build it locally using the following command:
+```bash
+docker build -t lidofinance/oracle .
+```
+
+## Run
+1. Use `.env.example` file content to create your own `.env` file.  
+      There are two options to run Oracle:
+   - `dry mode` (by default)
+   - `production mode` 
+\
+\
+      To run Oracle in `production mode` set `MEMBER_PRIV_KEY` environment variable in `.env` file:
+      ```bash
+      MEMBER_PRIV_KEY={value}
+      ```
+      Where `{value}` is a private key of the Oracle member account.
+2. Run the container using the following command:
+      ```bash
+      docker run --env-file .env lidofinance/oracle:{tag} {type}
+      ```
+      Where
+   - `{tag}` is a version of the image. You can find the latest version in the [releases](https://github.com/lidofinance/lido-oracle/releases)
+   - `{type}` is a type of the Oracle. There are two types of oracles:
+      - `accounting`
+      - `ejector`
+
+
+
+## Monitoring
+TBD
+
+### Dashboard
+TBD
+
+### Alerts
+TBD
+
+### Metrics
+TBD
+
 # Development
 
 Python version: 3.11
 
 ## Setup
 
 1. Setup poetry
+```bash
```

Comment:
Replace with link to poetry guide

Reply:
+ add table with variables

Reply:
https://github.com/lidofinance/depositor-bot#available-variables

Reply:
fixed